### PR TITLE
27571 d33 usage example

### PR DESCRIPTION
**Pull request by MathieuTillet**
**Description of work.**
Add an example to SANSILLAutoprocess documentation, plotting reduced data from D33.

**To test:**

Build documentation. Check the documentation for SANSILLAutoprocess now shows a correct plot as a new example.

Ref #27571

*This does not require release notes* because it is documentation.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by Harrietbrown** [comment]
I've been trying to build the docs but the plot code is failing, there seems to be an issue with the data used not being downloaded

**Comment by gvardany** [comment]
Did you try to run cmake and build the `StandardTestData` target before?

File: docs/source/algorithms/SANSILLAutoProcess-v1.rst [hunk 1]
```diff
@@ -87,6 +87,67 @@ Output:
...
+
+    # Plot the result of every autoprocess
+    for wName in sample_names:
+        ax.errorbar(mtd[wName][0])
+
+    plt.legend(sample_legends)
+    ax.set_ylabel('d$\sigma$/d$\Omega$ ($cm^{-1}$)')
+
+    #fig.show()
+    mtd.clear()
```
Comment (gvardany) [inline docs/source/algorithms/SANSILLAutoProcess-v1.rst]: I would remove the clear, since it might confuse the user running the example.

**Comment by Harrietbrown** [comment]
I forgot to regenorate in cmake. Now I am getting the data, I am getting errors while generating

5>SANSILLAutoProcess-[Notice] SANSILLAutoProcess started
5>SANSILLReduction-[Notice] SANSILLReduction started (child)
5>LoadAndMerge-[Notice] LoadAndMerge started (child)
5>LoadILLSANS-[Notice] LoadILLSANS started (child)
5>LoadILLSANS-[Error] Error in execution of algorithm LoadILLSANS:
5>LoadILLSANS-[Error] Error opening data in group "name"
5>LoadAndMerge-[Error] Error in execution of algorithm LoadAndMerge:
5>LoadAndMerge-[Error] Error opening data in group "name"
5>LoadAndMerge-[Error]   at line 123 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>LoadAndMerge-[Error]   caused by line 72 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>SANSILLReduction-[Error] Error in execution of algorithm SANSILLReduction:
5>SANSILLReduction-[Error] Error opening data in group "name"
5>SANSILLReduction-[Error]   at line 123 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>SANSILLReduction-[Error]   caused by line 72 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>SANSILLReduction-[Error]   at line 494 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\WorkflowAlgorithms\SANSILLReduction.py'
5>SANSILLReduction-[Error]   caused by line 1132 in 'c:\users\wey38795\documents\mantid_builds\master\raw\mantid\framework\pythoninterface\mantid\simpleapi.py'
5>SANSILLAutoProcess-[Error] Error in execution of algorithm SANSILLAutoProcess:
5>SANSILLAutoProcess-[Error] Error opening data in group "name"
5>SANSILLAutoProcess-[Error]   at line 123 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>SANSILLAutoProcess-[Error]   caused by line 72 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\LoadAndMerge.py'
5>SANSILLAutoProcess-[Error]   at line 494 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\WorkflowAlgorithms\SANSILLReduction.py'
5>SANSILLAutoProcess-[Error]   caused by line 1132 in 'c:\users\wey38795\documents\mantid_builds\master\raw\mantid\framework\pythoninterface\mantid\simpleapi.py'
5>SANSILLAutoProcess-[Error]   at line 302 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\WorkflowAlgorithms\SANSILLAutoProcess.py'
5>SANSILLAutoProcess-[Error]   caused by line 332 in 'C:/Users/wey38795/Documents/mantid_builds/Master/raw/mantid/Framework/PythonInterface/plugins/algorithms\WorkflowAlgorithms\SANSILLAutoProcess.py'
5>SANSILLAutoProcess-[Error]   caused by line 1132 in 'c:\users\wey38795\documents\mantid_builds\master\raw\mantid\framework\pythoninterface\mantid\simpleapi.py'

**Comment by MathieuTillet** [comment]
Have you tried rebuilding `SystemTestData` ? And checked that the path to ExternalData/Testing/Data/SystemTest is in  your data search directories?

**Comment by Harrietbrown** [comment]
I've rebuilt `SystemTestData`, and it is in my search directory. still no joy.

**Comment by gvardany** [comment]
@Harrietbrown Can you post the list of your data search directories?

**Comment by Harrietbrown** [comment]
.../Build/bin/Data/
.../Build/Testing/Data/SystemTest/
.../Build/ExternalData/Testing/Data/SystemTest/LOQ/
.../Build/ExternalData/Testing/Data/SystemTest/SAN2D/
.../Build/ExternalData/Testing/Data/SystemTest/LARMOR/
.../Build/ExternalData/Testing/Data/SystemTest/INTER/
.../Build/ExternalData/Testing/SystemTests/tests/analysis/reference/
.../Build/ExternalData/Testing/Data/UnitTest/
.../Build/ExternalData/Testing/Data/DocTest/
.../Build/Testing/SystemTests/scripts/
.../Build/bin/Data/SampleData-ISIS/loqdemo

**Comment by gvardany** [comment]
And what happens if you add also 
```.../Build/ExternalData/Testing/Data/SystemTest/```?

**Comment by Harrietbrown** [comment]
Oh I misstyped, that one is on my list.

**Comment by gvardany** [comment]
@Harrietbrown We are puzzled with your error message. Any chance you can try on linux? It looks like a windows only issue.

**Comment by Harrietbrown** [comment]
I found the issue, It wasn't windows, It was because my build had its facility set as ISIS and not ILL.
I don't know how the release build will manage this. I was able to build the doc it was just the plot part of it that failed, so in the release it may end up the same depending on how the build is set up.

**Comment by gvardany** [comment]
Ahh. I see the problem. Can you try with `config.setFacility('ILL')` instead of 
 this line ? https://github.com/mantidproject/mantid/pull/27584/files#diff-9d0f73315fa237c1418959a7d2315d2aR99

**Comment by Harrietbrown** [comment]
Yes that works and it doesn't change the facility on my build as well.

**Comment by gvardany** [comment]
@MathieuTillet Can you update the line? It seems we are missing the correct way of setting the facility after a recent change in config service api.

**Review by Harrietbrown — APPROVED** [review]
That works for me now. It's ready to go.

**Pull request merged**